doc/website/concepts/qos-policies.md

```diff
@@ -0,0 +1,33 @@
+# QoS policies
+
+Unlike other middlewares, e.g. [DDS used in ROS 2](https://docs.ros.org/en/galactic/Concepts/About-Quality-of-Service-Settings.html),
+iceoryx keeps things simple and offers basic Quality of Service (QoS) settings. The QoS
+settings are called options and can be used to optimize and tailor the communication.
+
+## `PublisherOptions` and `SubscriberOptions`
+
+The three most important settings are:
+
+| Option                              | Explanation                                                                                                                    |
+|-------------------------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| `PublisherOptions::historyCapacity` | This enables late-joining subscribers to request the n last samples                                                            |
+| `SubscriberOptions::queueCapacity`  | Size of the subscriber queue where samples are stored before they are passed to the user                                       |
+| `SubscriberOptions::historyRequest` | The number of samples a late-joining subscriber will request from a publisher, must be equal or smaller than `historyCapacity` |
```

Comment:
I'm not quite sure if it really must be smaller since the subscriber might not know the `historyCapacity`. I think if will silently accept if the request is larger and only delivers the available samples, similar to when the publisher is started and does not yet have enough samples.

I'm wondering what happens when the `historyRequest` is larger than the `queueCapacity`?

Reply:
Correct, it silently accepts it, but there is a warning in RouDi:

```
[Warning]: Chunk history request exceeds history capacity! Request is 20. Capacity is 10.
```

Re-phrased to "should be".